app/src/main/cpp/skyline/nce.cpp

```diff
@@ -168,20 +209,26 @@ namespace skyline {
     }
 
     std::vector<u32> NCE::PatchCode(std::vector<u8> &code, u64 baseAddress, i64 offset) {
```

Comment:
Might be an idea to split out the code patching to a new file, it's fairly large and we will might need to expand it in the future.

Reply:
I don't think that's required, the size of it isn't outrageous and I heavily doubt it'll be expanded in the future as there isn't much I can think of that can be doen in addition to what we already do.

Reply:
I think that it doesn't really fit with the whole 'theme' of the file, most of nce is related to execution not patching. there are also cases where we may want to hook specific functions.

Reply:
I feel like it totally does match the rest of the file as the rest of the file is essentially dealing with what the patcher has put in it. 
I'm marking this as resolved as I think it is of not much consequence and if the function's length is excessive then we might consider putting it in it's own file (Albeit in the NCE class/folder).

Reply:
Yeah, the nce file deals with what the patcher put in so logically the patching should go in a separate file in the nce folder.